lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -173,6 +173,7 @@ def authorize(amount, payment_method, options = {})
         add_account_name_inquiry(post, options)
         add_processor(post, options)
         add_authorization_details(post, options)
+        add_transaction_type(post, options)
```

Comment:
Just realized based on our conversation in the other Credorax PR that this `add_transaction_type` addition isn't necessary since we have the stored credential method above!